mass_mailing_custom_unsubscribe_event/tests/test_ui.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2020 Tecnativa - João Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import mock
+from werkzeug import urls
+
+from odoo.tests.common import HttpCase
+
+
+class UICase(HttpCase):
+    _tour_run = "odoo.__DEBUG__.services['web_tour.tour'].run('%s')"
+    _tour_ready = "odoo.__DEBUG__.services['web_tour.tour'].tours.%s.ready"
```

Comment:
You don't need this! :tada: 

See https://github.com/odoo/odoo/blob/980ecf1c1ca6f5f0cddab1065fbc669d6925ab67/odoo/tests/common.py#L1253-L1265